src/components/Sections/SectionsProgress.tsx

```diff
@@ -22,19 +34,79 @@ const SectionsProgress: React.FC<SectionsProgressProps> = ({
     )
   }
 
-  const getProgress = ({ doneRequired, doneNonRequired, totalSum, valid }: ProgressType) => {
+  const getIsStrictSection = (sectionCode: string) =>
+    sectionCode === firstStrictInvalidPage?.sectionCode
+
+  const setIsAfterStrict = (sectionCode: string) => {
```

Comment:
I am not a big fan of doing setIsAfterSrtict.

Would just do:
```
isStrictSectoin = ...
isAfterStrict = ...
```
 and use that scoped variable in `SectionAction`. 

Since the methods are scoped all of the props are available.

Reply:
No problem. That's tidier, will do that

Reply:
Hmmm not sure how to do that Andrei, I was using that as a set since all sections are in order, all sections that are after the `strictInvalid` would be set to all sections after that one (without having to check the index - which isn't passed). So not sure how to do that without a set.

Reply:
Done: 947ad03

Reply:
Oops sorry Nicole, that was a bad comment from me, the way you had it was good.

Your new changes should also work nicely